<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поставить напоминание</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #fff
}

#addBtn {
	background-color: #0D6EFD;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 18px;
}

#addBtn:hover {
	background-color: #0B5ED7;
}

#addBtn:pressed {
	background-color: #0B5ED7;
}

QTimeEdit::down-arrow {
	image: url(images/drop-down-list.png);
}

QTimeEdit::up-arrow {
	image: url(images/drop-down-list-toggle.png);
}

QTimeEdit::up-button {
	border-left: 1px solid #ccc;
	border-bottom: 1px solid #ccc;
}

QTimeEdit::down-button {
	border-left: 1px solid #ccc;
}

#deadlineCheckBox {
	background-color: #fff
}

#cancelBtn {
	background-color: #999;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 18px;
}

#cancelBtn:hover {
	background-color: #777;
}

#cancelBtn:pressed {
	background-color: #555;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-size: 24px</string>
        </property>
        <property name="text">
         <string>Поставить напоминание</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCalendarWidget" name="dateInput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>230</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="descriptionInput">
          <property name="placeholderText">
           <string>Описание...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>Выберите время:</string>
            </property>
            <property name="indent">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeInput">
            <property name="styleSheet">
             <string notr="true">font-size: 28px</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="text">
           <string>Поставить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
